### ISSUE TITLE
ACS URL undefined when using IdP other than the Onelogin example

### ISSUE THREAD

**Issue by fredpedro** [tngan/express-saml2#30]
Every IdP I test always return a "No return endpoint available for relying party express-saml2" error. Though I made sure the the AssertionConsumerService tag has an attribute of "Location" pointing to "http://localhost:3000/sso/acs".

**Comment by tngan** [comment]
@fredpedro Do you need to load any SP metadata into your IdP ? Also make sure it supports localhost access and not https. In fact, it would be good to provide more information and even make a sample repo, so I can help more.

**Comment by fredpedro** [comment]
Hi @tngan , here is the sample repository based from the application that I am working on. https://github.com/fredpedro/potential-robot. This is an Angular 2 application, but I removed the Angular 2 codes from the "client" folder. Yes, I need to load the SP metadata to our Gluu Server. I also tested the package against testshib.org's IdP but it raised the same error. If our Gluu Server is https enabled, does that mean I have to enable https on our Node.js app as well?
